modules/topic_space/main.tf

```diff
@@ -0,0 +1,11 @@
+resource "azapi_resource" "topic_space" {
```

Comment:
Please add `response_export_values = []` or be explicit abut the outputs you need in this attribute. This reduces the plan noise for module consumers.